ports/servoshell/prefs.rs

```diff
@@ -428,7 +428,11 @@ pub(crate) fn parse_command_line_arguments(args: Vec<String>) -> ArgumentParsing
     #[cfg(target_env = "ohos")]
     let log_filter = {
         let filters = opt_match.opt_strs("log-filter").join(",");
-        (!filters.is_empty()).then_some(filters)
+        let log_filter = (!filters.is_empty()).then_some(filters).or_else(|| {
```

Comment:
Was there a reason why we don't apply `--log-filter` or this pref to other target platforms if set?